python/paddle/fluid/tests/book/test_fit_a_line.py

```diff
@@ -70,11 +70,7 @@ def train_loop(main_program):
         train_loop(fluid.default_main_program())
     else:
         port = os.getenv("PADDLE_INIT_PORT", "6174")
-        pserver_ips = os.getenv("PADDLE_INIT_PSERVERS")  # ip,ip...
```

Comment:
Well, using `PADDLE_INIT_PSERVERS` is trying to be compatible to v2 and paddle cloud. If users need to run the test on one node using different ports to test multiple pservers and multiple trainers, they should modify the code then.

Or if we want to make these demos more simple and contain less code, can we just write a static configuration here.

Reply:
I'm not sure we need to compatible v2, and we have independent demo code on cloud repo. And, if we want to do the regression testing with distributed training on CI, these codes would running on one host.